### ISSUE TITLE
Remove confidence bands option using cal_plot_multi

### ISSUE THREAD

**Issue by drjgauthier** [ML4LHS/runway#4]
Things get a little crowded when comparing multiple models. 
For example:

<img src="https://user-images.githubusercontent.com/40614340/101198383-3d455100-3618-11eb-98ca-e18a05d1107b.png" width="300" height="300">

It would be very helpful to have the option to hide the confidence bands. Just a thought! ;)

**Comment by kdpsingh** [comment]
I agree we should provide this option, particularly for any of the multi-model plots where the overlapping confidence bands can get really busy. The default will still be to display the confidence intervals.

**Comment by kdpsingh** [comment]
Also, recognizing that journals often charge for color figures, we should consider showing difference models as different line types (eg dotted, dashed) if the confidence intervals don't need to be plotted.
